test/ziggurat/fixtures.clj

```diff
@@ -31,6 +33,17 @@
   (f)
   (mount/stop))
 
+(defn mount-tracer []
+  (with-redefs [tracer/create-tracer (fn [] (MockTracer.))]
```

Comment:
instead of a mockTracer why can we not have a noOp tracer here?

Reply:
mockTracer provides utility methods to check the number of spans completed in the test and so on. hence it is used to test the spans and corresponding tags